### PR TITLE
test: invalid and 404 vendor settings

### PR DESCRIPTION
**Pull request by danielgent**
This actually worked :smile:

### CODE REVIEW

File: test/integration/login/__image_snapshots__/vendor-settings-spec-ts-test-integration-login-vendor-settings-spec-ts-vendor-settings-should-fallback-to-sesamy-styling-with-invalid-vendor-id-1-snap.png
Comment (danielgent) [inline test/integration/login/__image_snapshots__/vendor-settings-spec-ts-test-integration-login-vendor-settings-spec-ts-vendor-settings-should-fallback-to-sesamy-styling-with-invalid-vendor-id-1-snap.png:1]: this is correct! we still get something rendered (new tests are below)

File: test/integration/login/__image_snapshots__/vendor-settings-spec-ts-test-integration-login-vendor-settings-spec-ts-vendor-settings-should-fallback-to-sesamy-styling-with-invalid-vendor-settings-response-1-snap.png
Comment (danielgent) [inline test/integration/login/__image_snapshots__/vendor-settings-spec-ts-test-integration-login-vendor-settings-spec-ts-vendor-settings-should-fallback-to-sesamy-styling-with-invalid-vendor-settings-response-1-snap.png:1]: same here! as expected :+1:

File: test/integration/login/vendor-settings.spec.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  it("should fallback to sesamy styling with invalid vendor settings response", async () => {
+    const env = await getEnv();
+    const oauthClient = testClient(oauthApp, env);
+
+    const response = await oauthClient.authorize.$get({
+      query: {
+        client_id: "clientId",
+        // bad vendor! in MSW this is a garbage object
+        vendor_id: "bad-vendor",
```
Comment (danielgent) [inline test/integration/login/vendor-settings.spec.ts:50]: :eyes:

**Review by markusahlstrand — APPROVED** [review]
LGTM

**Pull request merged**